web/modules/custom/dpl_fbi/src/Plugin/Field/FieldType/WorkIdItem.php

```diff
@@ -47,6 +47,10 @@ public static function propertyDefinitions(FieldStorageDefinitionInterface $fiel
       ->setLabel(t('Text value'))
       ->setRequired(TRUE);
 
+    $properties['material_type'] = DataDefinition::create('string')
+      ->setLabel(t('The material type'))
+      ->setRequired(TRUE);
```

Comment:
Consider if `setRequired()` should be `TRUE` here.

In general I do not think it should be required for editors to provide a material type on work fields.
I also wonder how this works in conjuction with us setting `material_type` as not null.